engines/config-query-sparql/config/config-default.json

```diff
@@ -3,59 +3,7 @@
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/config-query-sparql/^3.0.0/components/context.jsonld"
   ],
   "import": [
-    "ccqs:config/context-preprocess/actors.json",
-    "ccqs:config/context-preprocess/mediators.json",
-    "ccqs:config/hash-bindings/actors.json",
-    "ccqs:config/hash-bindings/mediators.json",
-    "ccqs:config/http/actors.json",
-    "ccqs:config/http/mediators.json",
-    "ccqs:config/http-invalidate/actors.json",
-    "ccqs:config/http-invalidate/mediators.json",
-    "ccqs:config/init/actors.json",
-    "ccqs:config/merge-bindings-context/actors.json",
-    "ccqs:config/merge-bindings-context/mediators.json",
-    "ccqs:config/optimize-query-operation/actors.json",
-    "ccqs:config/optimize-query-operation/mediators.json",
-    "ccqs:config/query-operation/actors.json",
-    "ccqs:config/query-operation/mediators.json",
-    "ccqs:config/query-parse/actors.json",
-    "ccqs:config/query-parse/mediators.json",
-    "ccqs:config/query-process/actors.json",
-    "ccqs:config/query-process/mediators.json",
-    "ccqs:config/query-result-serialize/actors.json",
-    "ccqs:config/query-result-serialize/mediators.json",
-    "ccqs:config/query-source-identify/actors.json",
-    "ccqs:config/query-source-identify/mediators.json",
-    "ccqs:config/query-source-identify-hypermedia/actors.json",
-    "ccqs:config/query-source-identify-hypermedia/mediators.json",
-    "ccqs:config/dereference/actors.json",
-    "ccqs:config/dereference/mediators.json",
-    "ccqs:config/dereference-rdf/actors.json",
-    "ccqs:config/dereference-rdf/mediators.json",
-    "ccqs:config/rdf-join/actors.json",
-    "ccqs:config/rdf-join/mediators.json",
-    "ccqs:config/rdf-join-entries-sort/actors.json",
-    "ccqs:config/rdf-join-entries-sort/mediators.json",
-    "ccqs:config/rdf-join-selectivity/actors.json",
-    "ccqs:config/rdf-join-selectivity/mediators.json",
-    "ccqs:config/rdf-metadata/actors.json",
-    "ccqs:config/rdf-metadata/mediators.json",
-    "ccqs:config/rdf-metadata-accumulate/actors.json",
-    "ccqs:config/rdf-metadata-accumulate/mediators.json",
-    "ccqs:config/rdf-metadata-extract/actors.json",
-    "ccqs:config/rdf-metadata-extract/mediators.json",
-    "ccqs:config/rdf-parse/actors.json",
-    "ccqs:config/rdf-parse/mediators.json",
-    "ccqs:config/rdf-parse-html/actors.json",
-    "ccqs:config/rdf-resolve-hypermedia-links/actors.json",
-    "ccqs:config/rdf-resolve-hypermedia-links/mediators.json",
-    "ccqs:config/rdf-resolve-hypermedia-links-queue/actors.json",
-    "ccqs:config/rdf-resolve-hypermedia-links-queue/mediators.json",
-    "ccqs:config/rdf-serialize/actors.json",
-    "ccqs:config/rdf-serialize/mediators.json",
-    "ccqs:config/rdf-update-hypermedia/actors.json",
-    "ccqs:config/rdf-update-hypermedia/mediators.json",
-    "ccqs:config/rdf-update-quads/actors.json",
-    "ccqs:config/rdf-update-quads/mediators.json"
+    "ccqs:config/config-base.json",
```

Comment:
See my comment in rdf-join, I don't think we need any changes to the `config-` files in this repo.